features/step_definitions/basic_steps.rb

```diff
@@ -36,3 +36,11 @@
   user = User.find_by(email: email)
   login_as(user, scope: :user)
 end
+
+Then(/^I am on the index booking page$/) do
+  visit bookings_path
+end
+
+Given(/^the following facility exist$/) do
+  create(:facility)
+end
```

Comment:
This test only passed because you were using `Facility.first` up in the controller and that is only going to work for one facility. You need to be able to handle lots of facilities.